src/Sylius/Bundle/CoreBundle/Form/DataTransformer/TaxonsToCodesTransformer.php

```diff
@@ -61,20 +61,25 @@ public function reverseTransform($value)
             throw new UnexpectedTypeException($value, Collection::class);
         }
 
-        if ($value->isEmpty() || null === $value->get('taxons')) {
+        $taxons = $value->get('taxons');
+
+        // ensure not empty / null
+        if ($value->isEmpty() || null === $taxons) {
```

Comment:
We don't know if `$taxons` is `ArrayCollection` instance. There is no need to check for empty.